crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -326,7 +326,7 @@ pub struct AuthorityEpochTables {
 
     /// JWKs that are currently available for zklogin authentication, and the round in which they
     /// became active.
-    active_jwks: DBMap<u64, (JwkId, JWK)>,
+    active_jwks: DBMap<(u64, (JwkId, JWK)), ()>,
```

Comment:
Does JWK become part of the key? That's quite non-intuitive. I don't have the context, but would be good to see a doc comment explanation at least.

Reply:
Yes, it's just storing a set of jwks with the round in which they became active, and its stored this way to make it easy to iterate by round.

The other option would be to have `DBMap<((u64, u64 /* index */), (JwkId, (JWK))>`, but this would require additional code to generate unique indices for each jwk.